src/test/java/core/basesyntax/FileReaderImplTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import core.basesyntax.service.FileReader;
+import core.basesyntax.service.impl.FileReaderImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static FileReader fileReader;
+    private static final String INPUT_FILE_PATH = "src/main/resources/input.csv";
+    private static final String EMPTY_FILE_PATH = "src/main/resources/empty.csv";
+    private static final String FAKE_FILE_PATH = "src/main/***/resources/input.csv";
+
+    @BeforeClass
+    public static void setUp() {
+        fileReader = new FileReaderImpl();
+    }
+
+    @Test
+    public void read_FromFile_Ok() {
```

Comment:
```suggestion
    public void read_fromFile_ok() {
```
the first letter after ```_``` in lowerCase (check it everywhere)